sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -4976,8 +4976,9 @@ pi_result piextKernelSetArgMemObj(pi_kernel Kernel, pi_uint32 ArgIndex,
   //       piextKernelSetArgMemObj.
   //
   std::scoped_lock Guard(Kernel->Mutex);
+  auto Arg = ArgValue ? *ArgValue : nullptr;
```

Comment:
Please add a comment similar to what is at line 4936

Reply:
Added here: f8bca296e2fb5dc8f48a079a7dc6cabb66ca1193. Thank you.